build.gradle

```diff
@@ -33,6 +35,17 @@ googleJavaFormat {
   exclude 'build/**'
 }
 
+if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
+    project.name != 'benchmark') {
```

Comment:
It's possible we might add more projects. I think we should create a list of project names and do "project.name in list" instead. (I don't actually know if the "in" syntax actually works in Groovy or not).